typings/react-dropzone.d.ts

```diff
@@ -66,8 +66,8 @@ export type DropzoneState = DropzoneRef & {
   isDragAccept: boolean;
   isDragReject: boolean;
   isFileDialogActive: boolean;
-  draggedFiles: File[];
-  acceptedFiles: File[];
+  draggedFiles: (File & { path?: string })[];
+  acceptedFiles: (File & { path?: string })[];
```

Comment:
Isn't `FileWithPath` the same thing?
```suggestion
  draggedFiles: FileWithPath[];
  acceptedFiles: FileWithPath[];
```
Could you try that and see if you still get errors?

Reply:
I always see this error, see [this demo](https://github.com/aslan-alt/react-dropzone-ts-demo)

Reply:
@aslan-alt could you update to v14 and see if the same issue happens? I'll take a look later this week.

Reply:
I just tried it out and if you set:
```
 acceptedFiles: FileWithPath[];
```
as suggested, the error is gone.

So please update that and also set the `FileRejection.file` prop type to the same type.

Reply:
okay